src/ir/comment.rs

```diff
@@ -38,6 +38,38 @@ fn make_indent(indent: usize) -> String {
     iter::repeat(' ').take(indent * RUST_INDENTATION).collect()
 }
 
+/// Preprocesses comments containing brackets to prevent `unresolved link to ...`
+///
+/// Avoid processing expensive computations if no brackets are found
+fn preprocess_plain_brackets(line: &str) -> String {
+    if line.contains("[") && line.contains("]") {
+        line.chars()
```

Comment:
I kinda feel this would be more readable as a regular for loop, but maybe it's just me so if you feel strongly against that no big deal :)